content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -222,6 +224,70 @@ At this point, the flow is now complete.
    order to confirm the delivery. Otherwise, the delivered quantity can be entered manually directly
    on the sales order.
 
+.. _sales/invoicing/down_payment/100-percent-down-payments:
+
+Example: request 100% down payment
+==================================
+
+The process of requesting a 100% down payment is similar to the process of setting up a :ref:`50%
+down payment <sales/invoicing/down_payment/50-percent-down-payments>` but with fewer steps. The
+*Solar Panel Installation* product is being used in this example.
+
+To configure a 100% down payment, begin by navigating to :menuselection:`Sales app --> New` and add
+a :guilabel:`Customer` to the quote.
+
+Next, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
+:guilabel:`Solar Panel Installation` product.
+
+:guilabel:`Confirm` the order and the quotation will turn into a sales order. An invoice can now be
+created by clicking :guilabel:`Create Invoice` in the top-left.
+
+.. image:: down_payment/100p-order-confirmed.png
+   :align: center
+   :alt: Confirmed sales order for the Solar Panel Installation product.
+
+On the :guilabel:`Create invoices` pop-up window that appears, select :guilabel:`Down payment
+(percentage)`, and type `100` in the :guilabel:`Down Payment Amount` field.
```

Comment:
```suggestion
(percentage)`, and type `100` in the :guilabel:`Down Payment Amount` field. Then, if desired, select an :guilabel:`Income Account' and :guilabel:`Customer Taxes` in their respective fields.
```
Just some added information to round out the fields mentioned in the 'Create invoice' pop-up window